components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/util/OIDCErrorConstants.java

```diff
@@ -51,7 +71,23 @@
         REQUESTING_ACCESS_TOKEN_FAILED("OID-65006",
                 "Exception while requesting access token"),
         EXECUTING_CLAIM_TRANSFORMATION_FOR_IDP_FAILED("OID-65007",
-                "Error while executing claim transformation for IDP: %s");
+                "Error while executing claim transformation for IDP: %s"),
+        // Federated IdP initiated back-channel logout server errors.
+        FEDERATED_SESSION_TERMINATION_FAILED("OID-65008",
+                "Unable to terminate session for session Id: %s"),
+        RETRIEVING_SESSION_ID_MAPPING_FAILED("OID-65009",
+                "Error while retrieving session Id mapping for sid: %s"),
+        RETRIEVING_IDENTITY_PROVIDER_FAILED("OID-65010",
+                "Error while retrieving the identity provider"),
+        NO_REGISTERED_IDP_FOR_ISSUER("OID-65011",
+                "No Registered IDP found for the issuer name: %s found in JWT"),
+        GETTING_RESIDENT_IDP_FAILED("OID-65012",
```

Comment:
Some errors have a full stop at the end while others don't have it.
Let's make it consistent